doc/Language/variables.pod6

```diff
@@ -411,8 +414,16 @@ Placeholder variables cannot have type constraints or a variable name with a
 single upper-case letter (this is disallowed to enable catching some
 Perl-isms).
 
+The C<^> twigil can be combined with any sigil to create a placeholder variable
+with the appropriate type constraint.  Thus C<@^positional>,C<%^associative>,and
```

Comment:
Wouldn't you describe "positional" and "associative" as "roles" more than type constraints? After all, it can be any type that does that role.